src/converters/fromarray.ts

```diff
@@ -1,6 +1,6 @@
-import { ObjectType } from "../../types/index";
-import {  IEvent, IEventArray } from "../../types/index";
+import { IEvent, IEventArray, ObjectType } from "../../types/core";
```

Comment:
Where are these types/core files? Are they automatically generated? I don't see them in the PR.

Reply:
They all reside in the ./types folder. They aren't in the PR because i haven't touched them - just importing from different files

Reply:
They all reside in the ./types folder. They aren't in the PR because i haven't touched them - just importing from different files


---
In reply to: [250244704](https://github.com/Microsoft/clarity-js/pull/123#discussion_r250244704) [](ancestors = 250244704)

Reply:
So I thought it was weird that we had all of our typing files in the same folder and also an uber typing file that connected them all together. It looks like the standard is to keep our .d.ts files located with the code that implements them (which makes some sense to me, easy to go from the typing file to the implementation) and then our uber file would continue to grab from the different locations and export them from a single place. 

[https://github.com/Microsoft/TypeScript/issues/8305](https://github.com/Microsoft/TypeScript/issues/8305)

Thoughts?

---
In reply to: [250325003](https://github.com/Microsoft/clarity-js/pull/123#discussion_r250325003) [](ancestors = 250325003,250244704)

Reply:
Looks like there are some great ideas out there and perhaps better ways than what we are doing right now. I'm scoping it outside of this PR though, feel free to create an issue to reorganize typedefs

---
In reply to: [250692834](https://github.com/Microsoft/clarity-js/pull/123#discussion_r250692834) [](ancestors = 250692834,250325003,250244704)